package.json

```diff
@@ -11,7 +11,7 @@
     "dashboarding"
   ],
   "private": true,
-  "version": "2.3.0",
+  "version": "2.3.1",
   "branch": "main",
```

Comment:
NIT: can we bump line 15 to be `2.3` it's used for documentation. So this works to point to `latest` if it is defined as `main` for example https://opensearch.org/docs/latest/. But since `2.3.0` was released with a doc site we now have this https://opensearch.org/docs/2.3/

Reply:
Updated.

Reply:
Thank you so much!